app/src/test/java/io/github/jbellis/brokk/context/UndoRedoRestoreTest.java

```diff
@@ -0,0 +1,48 @@
+package io.github.jbellis.brokk.context;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.github.jbellis.brokk.analyzer.ProjectFile;
+import io.github.jbellis.brokk.testutil.NoOpConsoleIO;
+import io.github.jbellis.brokk.testutil.TestContextManager;
+import java.nio.file.Path;
+import java.util.List;
+import org.fife.ui.rsyntaxtextarea.SyntaxConstants;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class UndoRedoRestoreTest {
```

Comment:
This is migrated from https://github.com/BrokkAi/brokk/compare/master...contextunfreeze3?diff=split&w